libretro/libretro_core_options.h

```diff
@@ -304,8 +325,13 @@ struct retro_core_option_definition option_defs_us[] = {
             {"True", NULL},
             {NULL, NULL},
         },
+#ifdef HAVE_OPENGLES
+        "False
+#else
         "True"
```

Comment:
this is enabled by default on upstream - https://github.com/gonetz/GLideN64/blob/a96853d559680cca17fdc4dfa5891dad03d79b66/src/Config.cpp#L58 - safe? works on my GLES2 and 3 devices

Reply:
This is from https://github.com/libretro/mupen64plus-libretro-nx/blob/develop/libretro/libretro.c#L293-L294

Again I'd suggest a separate MR for changing behaviour.